### DISCUSSION TITLE
phpMyAdmin not selected in laragon

### DISCUSSION THREAD

**Discussion by tuhinrehan-125** [leokhoa/laragon#581] [Q&A]
In Laragon, when I select HeidiSQL it works and opens the window. But when I select phpMyAdmin it says something like that: "I regret to inform you that due to financial difficulties, I cannot keep the self-hosted forum and moving our discussions to GitHub.

https://github.com/leokhoa/laragon/discussions

I understand that this may cause inconvenience and I apologize for any disruption this may cause.

I hope to continue our discussions and exchange of ideas on GitHub and I look forward to your participation there.

If you wonder about the future of Laragon, I have made a story about "[How I Created Laragon and the Current Status of Laragon’s Development](https://medium.com/@leovn/how-i-created-laragon-and-the-current-status-of-laragons-development-bc927c5ce1d6)"

I appreciate your understanding and support."

So now, how can I solve this problem and get phpMyAdmin?

**Comment by sh4msi** [comment]
See https://github.com/leokhoa/laragon/discussions/516 and https://github.com/leokhoa/laragon/discussions/393
